tox.ini

```diff
@@ -81,7 +81,7 @@ deps =
 usedevelop = True
 commands =
     pip install -U {toxinidir}/tests/functional/gradient_descent_algo
-    python setup.py test --addopts '-vvv --exitfirst --capture=no --looponfail {posargs}'
```

Comment:
Why did you drop the --capture=no?

Reply:
If you recall, the `capture=no` led to a bug we had some months ago.